config/locales/en/coronavirus_landing_page.yml

```diff
@@ -23,34 +22,6 @@ en:
                 url: /government/publications/covid-19-stay-at-home-guidance/stay-at-home-guidance-for-households-with-possible-coronavirus-covid-19-infection
```

Comment:
Think this is just `/government/publications/covid-19-stay-at-home-guidance` now - so it's to the splash page, not the html attachment